packages/expo-build-properties/src/android.ts

```diff
@@ -207,3 +212,31 @@ export function updateAndroidProguardRules(
   }
   return newContents;
 }
+
+export const withAndroidCleartextTraffic: ConfigPlugin<PluginConfigType> = (config, props) => {
+  return withAndroidManifest(config, (config) => {
+    if (props.android?.usesCleartextTraffic == null) {
+      return config;
+    }
+
+    config.modResults = setUsesCleartextTraffic(
+      config.modResults,
+      props.android?.usesCleartextTraffic
+    );
+
+    return config;
+  });
+};
+
+function setUsesCleartextTraffic(
+  androidManifest: AndroidConfig.Manifest.AndroidManifest,
+  value: boolean
+) {
+  const mainApplication = getMainApplicationOrThrow(androidManifest);
+
+  if (mainApplication?.$) {
+    mainApplication.$['android:usesCleartextTraffic'] = String(value) as StringBoolean;
```

Comment:
```suggestion
    mainApplication.$['android:usesCleartextTraffic'] = String(
      value
    ) as AndroidConfig.Manifest.StringBoolean;
```

we can reuse the import from AndroidConfig and not import directly from @expo/config-plugins